### PR TITLE
Add UC4 to DeveloperGuide.md

### LINKED ISSUE #27: Add use cases for 'find' command
Add use cases for find command to developer's guide

### PR DESCRIPTION
**Pull request by pbthang**
Close #27

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -479,6 +478,32 @@ Use case ends
...
+- 1a. User requests to search for reservation availability at a date.
+    - 1a1. RHRH shows all the available slots on the indicated date.  
+      Use case ends.
+
+- 1b. User requests to search for reservation availability at a time.
+    - 1b1. RHRH shows the availability for the indicated time today.  
+      Use case ends.
+
+- 1c. RHRH detects information missing from the command entered.
+    - 1c1. RHRH requests user to enter the command again with sufficient information.
```
Comment (Javiier-pzk) [inline docs/DeveloperGuide.md:500]: Hi Thang, you can refer to  the google docs to see the format for the use case please follow what me and Chetwin wrote.
For this you should have another line 1c2 `user enters command again` or `user enters missing info`. then another line that days `Steps 1c1 - 1c2 are repeated until the command is correctly entered` and then another line that days `use case resumes from 1 /1a /1b` depending on what the user entered.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -479,6 +478,32 @@ Use case ends
...
+- 1b. User requests to search for reservation availability at a time.
+    - 1b1. RHRH shows the availability for the indicated time today.  
+      Use case ends.
+
+- 1c. RHRH detects information missing from the command entered.
+    - 1c1. RHRH requests user to enter the command again with sufficient information.  
+      Use case ends.
+
+- 1d. RHRH detects wrong format of date/time.
+    - 1d1. RHRH requests user to enter the command again with a suitable date/time format.
```
Comment (Javiier-pzk) [inline docs/DeveloperGuide.md:504]: Same for this

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (pbthang) [reply docs/DeveloperGuide.md]: Ok sorry, I fixed it.

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (pbthang) [reply docs/DeveloperGuide.md]: Fixed

**Review by Javiier-pzk — APPROVED** [review]
LGTM. Thanks for editing

**Pull request merged**